mep/people/management/commands/export_locations.py

```diff
@@ -0,0 +1,119 @@
+        "street_address",
+        "postal_code",
+        "city",
+        "arrrondissement",
+        "country",
+        "start_date",
+        "end_date",
+        "start_date_precision",
+        "end_date_precision",
+        "longitude",
+        "latitude",
+    ]
+
+    def get_queryset(self):
+        """
+        custom filter needed to return person-address combos,
+        so we can pass a one object per row to `get_object_data`
+        """
+        addresses = Address.objects.prefetch_related(
+            Prefetch("account"),
+            Prefetch("person"),
+            Prefetch("location"),
+        )
+        res = []
+        for addr in addresses.all():
+            persons = [addr.person] if addr.person else addr.account.persons.all()
+            for person in persons:
+                res.append((person, addr))
+        return res
+
+    def get_base_filename(self):
+        """set the filename to 'locations.csv'"""
+        return "locations"
+
+    def get_object_data(self, obj):
+        """
+        Generate dictionary of data to export for a single
+        :class:`~mep.people.models.Person`
+        """
+        person, addr = obj
+        loc = addr.location
+
+        # format id
+        id_parts = [
+            person.slug,
+            loc.street_address,
+            loc.city,
+            addr.start_date.isoformat() if addr.start_date else None,
+            addr.end_date.isoformat() if addr.end_date else None,
+        ]
+        id = "-".join(x for x in id_parts if x).lower()
+
+        # required properties
+        return dict(
+            # Member
+            id=id,
+            member_id=person.slug,
+            member_uri=absolutize_url(person.get_absolute_url()),
+            # Address data
+            start_date=addr.start_date.isoformat() if addr.start_date else None,
+            end_date=addr.end_date.isoformat() if addr.end_date else None,
+            start_date_precision=serialize_precision(addr.start_date_precision)
+            if addr.start_date_precision
+            else None,
+            end_date_precision=serialize_precision(addr.end_date_precision)
+            if addr.end_date_precision
+            else None,
+            care_of_person_id=addr.care_of_person.slug if addr.care_of_person else None,
+            # Location data
+            street_address=loc.street_address,
+            city=loc.city,
+            postal_code=loc.postal_code,
+            latitude=float(loc.latitude) if loc.latitude is not None else None,
+            longitude=float(loc.longitude) if loc.longitude is not None else None,
+            country=loc.country.name if loc.country else None,
+            arrrondissement=loc.arrondissement(),
+        )
+
+
+def serialize_precision(date_prec):
+    o = []
+    if date_prec.year:
+        o.append("Y")
+    if date_prec.month:
+        o.append("M")
+    if date_prec.day:
+        o.append("D")
+    return "-".join(o)
```

Comment:
should not be needed - refer to `mep.accounts.partial_date` for the existing logic

```suggestion
```